spec/requests/sms_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
```

Comment:
A+! love using using the request spec here to test the controller behavior

Reply:
What's funny is that after pontificating, this seemed like one case where a controller spec actually was a better option, since this endpoint doesn't have a true user-facing component that requires full-stack testing, and only really needs to test auth and response codes.